src/Relude/Foldable/Fold.hs

```diff
@@ -73,8 +73,8 @@ flipfoldl' f = foldl' (flip f)
 >>> asumMap (\x -> if x > 2 then Just x else Nothing) [1..4]
 Just 3
 -}
-asumMap :: (Foldable f, Alternative m) => (a -> m b) -> f a -> m b
-asumMap f = getAlt . foldMap (Alt . f)
+asumMap :: forall f m a b . (Foldable f, Alternative m) => (a -> m b) -> f a -> m b
```

Comment:
Let's change the order of variables here. For convenience reasons I propose the following order which seems more convenient to me
```suggestion
asumMap :: forall m b f a . (Foldable f, Alternative m) => (a -> m b) -> f a -> m b
```

Reply:
This would be a breaking change, as they're already implicitly in the order that I explicitly put them in. For example, `asumMap @[] (\x -> if x > 2 then Just x else Nothing) [1..4]` is valid today, but would be invalid if I made this change.

Reply:
It's okay that this is a breaking change because this change is going to the next major release. It's better to have a more useful type signature in this case. I don't expect a lot of people to specify type variables for this function. But good point about breaking change :+1: We provide breaking changes announcements in our changelogs and migration guides, so need to remember to do this before release for this particular feature.